web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -123,6 +132,67 @@ be edited, its `Revision: X` label will be increased by one, and a note will be
 to the `PDEP-N history` section. This will let readers understand that the PDEP has
 changed and avoid confusion.
 
+## <a id="List-of-issues"></a> List of issues that could have been PDEPs for context
+### Clear examples for potential PDEPs:
+- Adding a new parameter to many existing methods, or deprecating one in many places. For example:
+  - The `numeric_only` deprecation affected many methods and could have been a PDEP.
+- Adding a new data type has impact on a variety of places that need to handle the data type.
+  Such wide-ranging impact would require a PDEP. For example:
+  - `Categorical` ([GH-7217][7217], [GH-8074][8074]), `StringDtype` ([GH-8640][8640]), `ArrowDtype`
+- A significant (breaking) change in existing behavior. For example:
+  - Copy/view changes ([GH-36195][36195])
+- Support of new python features with a wide impact on the project. For example:
+  - Supporting typing within pandas vs. creation of `pandas-stubs` ([GH-43197][43197],
+    [GH-45253][45253])
+- New required dependency
+
+### Borderline examples:
+- Small changes to core functionality, such as `DataFrame` and `Series`, should always be considered
+  as a PDEP candidate as it will likely have a big impact on users. But the same types of
+  changes in other methods would not be good PDEP candidates. That said, any discussion, no
```

Comment:
```suggestion
  changes in other objects would not be good PDEP candidates. That said, any discussion, no
```

? (not fully sure what our intent was here, but I think the idea was that for example adding a new core method to DataFrame has wider impact than a new method on eg Timestamp)